src/lib/drivers/bm_dfu_wrapper.cpp

```diff
@@ -65,20 +64,6 @@ uint32_t bm_dfu_client_flash_area_get_size(const void *flash_area) {
   return flash_area_get_size((const struct flash_area *)flash_area);
 }
 
-bool bm_dfu_client_confirm_is_enabled(void) {
-  uint32_t val = 1;
-  get_config_uint(BM_CFG_PARTITION_SYSTEM, dfu_confirm_config_key,
-                  strlen(dfu_confirm_config_key), &val);
-  return val == 1;
-}
-
-void bm_dfu_client_confirm_enable(bool en) {
-  uint32_t val = en;
-  set_config_uint(BM_CFG_PARTITION_SYSTEM, dfu_confirm_config_key,
-                  strlen(dfu_confirm_config_key), val);
-  save_config(BM_CFG_PARTITION_SYSTEM, true);
-}
-
```

Comment:
these have been moved to the dfu_client.c file in `bm_core`. See https://github.com/bristlemouth/bm_core/pull/63